src/styles/_footer.scss

```diff
@@ -83,7 +78,11 @@
 /* || DESKTOP */
 
 @media (min-width: units('desktop')) {
-  .footer {
+  .footer__return-to-top-feedback-container {
```

Comment:
This seems empty, do we need it?

Reply:
I'm still working on the return to top placement so that's why it's empty for now